software/apps/audio_module/event_detector/Makefile

```diff
@@ -0,0 +1,17 @@
+# makefile for user application
+
+# the current directory
+APP_DIR := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
+
+# files needed for this code
+C_SRCS   := $(wildcard *.c) 
```

Comment:
Ugh... yes you are right... the problem is that there is no `lib2fix` subdirectory in the `build/cortex-m4` directory and our Makefile rule assumes there is (and GCC won't just create one). If you create one after the build fails the first time, it will be able to move past it, but that's not solution.

@ppannuto please halp! You are the resident dependency file expert!